src/js/shariff.js

```diff
@@ -112,6 +112,12 @@ Shariff.prototype = {
 
         flattrUser: null,
 
+        createLinkIcon: function($shareLink, service) {
+            if (typeof service.faName !== 'undefined') {
+                $shareLink.prepend('<span class="fa ' +  service.faName + '">');
```

Comment:
You should use `<i class="fa ' +  service.faName + '" aria-hidden="true"></i>` instead of the old span tag